### PR TITLE
[MIG][website_crm_privacy_policy] Migrate to v10

### PR DESCRIPTION
**Pull request by yajo**
Very simple migration.

@Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you rebase to see if sample pages work?

File: website_crm_privacy_policy/templates/contactus.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            <div class="col-md-7 col-md-offset-3 col-sm-8 col-sm-offset-4">
+                <div class="checkbox">
+                    <input type="checkbox"
+                        value="accepted"
+                        id="privacy_policy"
+                        name="privacy_policy"
+                        class="o_website_form_input"
+                        required="required"/>
+                    <label class="control-label">
+                        <t t-call="website_legal_page.acceptance_full"/>
```
Comment (chienandalu) [inline website_crm_privacy_policy/templates/contactus.xml:20]: The default links given here aren't valid. For example: `/page/terms` where the default terms page is on `/legal/terms-of-use`

**Comment by yajo** [comment]
Rebased, should work now

**Review by chienandalu — APPROVED** [review]
Perfect. Now it works :slightly_smiling_face: :+1:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**